src/govuk/objects/objects.test.js

```diff
@@ -1,11 +1,58 @@
 /* eslint-env jest */
 
+const outdent = require('outdent')
+
 const glob = require('glob')
 const { renderSass } = require('../../../lib/jest-helpers')
 const configPaths = require('../../../config/paths.json')
 
+const sassConfig = {
+  outputStyle: 'nested'
+}
+
 const sassFiles = glob.sync(`${configPaths.src}/objects/**/*.scss`)
 
-it.each(sassFiles)('%s renders to CSS without errors', (file) => {
-  return renderSass({ file: file })
+describe('The objects layer ', () => {
+  describe('@mixin govuk-width-container', () => {
```

Comment:
Ideally I think this should be in its own file – `width-container.test.js`. In the other layers, we only put tests that apply to the whole layer in the layer test file.

Reply:
Yes that's a good point, I've moved it 👍